### PR TITLE
Added option for change dns

### PR DESCRIPTION
**Pull request by padsalatushal**
Added a dropdown menu box for choosing  DNS like google, cloud flare, level3, and OpenDNS. After selecting when the run tweak button press changes DNS for all connations. The default value is a default that does not change anything.

### CODE REVIEW

File: winutil.ps1 [hunk 1]
```diff
@@ -909,6 +909,39 @@ $WPFtweaksbutton.Add_Click({
...
         }
+        If ( $WPFchangedns.text -eq 'Google' ) { 
+            Write-Host "Setting DNS to Google for all connections..."
+            $DC = "8.8.8.8"
+            $Internet = "8.8.4.4"
+            $dns = "$DC", "$Internet"
+            $Interface = Get-WmiObject Win32_NeworkAdapterConfiguration 
+            $Interface.SettDNSServerSearchOrder($dns)  | Out-Null
+        }
+        If ( $WPFchangedns.text -eq 'Cloud Fair' ) {
```
Comment (Sp5rky) [inline winutil.ps1:934]: It's CloudFlare, not Fair

File: MainWindow.xaml [hunk 2]
```diff
@@ -237,6 +237,14 @@
 							<CheckBox Name="EssTweaksRemoveCortana" Content="Remove Cortana" Margin="5,0" ToolTip="Removes Cortana, but often breaks search... if you are a heavy windows search users, this is NOT recommended."/>
 							<CheckBox Name="EssTweaksRemoveEdge" Content="Remove Microsoft Edge" Margin="5,0" ToolTip="Removes MS Edge when it gets reinstalled by updates."/>
 							<CheckBox Name="MiscTweaksRightClickMenu" Content="Set Classic Right-Click Menu " Margin="5,0" ToolTip="Great Windows 11 tweak to bring back good context menus when right clicking things in explorer."/>
+							<Label Content="DNS" />
+							<ComboBox Name="changedns"  Height = "20" Width = "100" HorizontalAlignment = "Left" > 
+								<ComboBoxItem IsSelected="True" Content = "Default"/> 
+								<ComboBoxItem Content = "Google"/> 
+								<ComboBoxItem Content = "Cloud Fair"/>
```
Comment (Sp5rky) [inline MainWindow.xaml:244]: It's CloudFlare, not Fair

**Comment by ChrisTitusTech** [comment]
Love this, I just rebased the project, so we will have to redo it with a new test branch I'll push out later today

**Comment by padsalatushal** [comment]
@ChrisTitusTech now can you reopen this?

**Comment by padsalatushal** [comment]
@ChrisTitusTech should I create a new pr for resolving conflicts?

**Comment by ChrisTitusTech** [comment]
Probably be a lot easier

**Pull request closed without merging**